go/pkg/metastore/db/dao/record_log.go

```diff
@@ -28,20 +28,22 @@ func (s *recordLogDb) PushLogs(collectionID types.UniqueID, recordsContent [][]b
 			zap.Int("count", len(recordsContent)))
 
 		var lastLog *dbmodel.RecordLog
-		err := tx.Select("id").Where("collection_id = ?", collectionIDStr).Last(&lastLog).Error
+		err := tx.Select("log_offset").Where("collection_id = ?", collectionIDStr).Order("log_offset DESC").Limit(1).Find(&lastLog).Error
```

Comment:
This is a real change - we have to do this because goorm writes the query with .id as the sort key by default if there is a composite key. We specificy explicitly here

Reply:
LGTM